ttnn/cpp/ttnn/deprecated/tt_dnn/op_library/optimizer/optimizer_ops.cpp

```diff
@@ -47,7 +47,7 @@ std::vector<Tensor> _lamb_optimizer(const Tensor& data, const Tensor& grad, cons
         return data_val;
     };
     Tensor data_val = rmsnorm(data);
-    Tensor weight_norm = clamp(data_val, 0.0f, 10.0f, output_mem_config);
+    Tensor weight_norm = ttnn::operations::unary::_clamp(data_val, 0.0f, 10.0f, output_mem_config);
```

Comment:
Why are we not using the registered ops?

Reply:
Hi @eyonland , we tried to use registered function like `ttnn::clamp`, but by doing so, we are getting redefinition error for few ops like tril, triu as those ops are not completely migrated to TTNN (done in the PR #10437), Once it goes to main, we can modify this code as you suggested

Reply:
Not good. Should use registered operation. We should see whats going on, why you get redefinition issue and resolve that.

Reply:
please show the error. We definitely can fix it right here.

Reply:
We tried including the header `ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp` here as this contains the op registrations and called the op like `ttnn::clamp` , But faced redeclaration issue as below
![Screenshot 2024-07-25 at 6 42 26 PM](https://github.com/user-attachments/assets/86244a4e-90a2-4ac7-a340-b4a91130ad47)

Reply:
Lets merge #10473 and fix it here after/